cli/js/lib.deno_runtime.d.ts

```diff
@@ -170,7 +172,7 @@ declare namespace Deno {
    * | macOS   | `$HOME`/Movies      | /Users/Alice/Movies   |
    * | Windows | `{FOLDERID_Videos}` | C:\Users\Alice\Videos |
    */
-  export function dir(kind: DirKind): string;
+  export function dir(kind: DirKind, fallback?: string): string;
```

Comment:
Eh this doesn’t belong in the dir. I’m fine with returning null on error instead of throwing.

Reply:
Oh, that's even better!